server/routes/induction/common/personalInterestsController.ts

```diff
@@ -0,0 +1,27 @@
+import { NextFunction, Request, RequestHandler, Response } from 'express'
+import InductionController from './inductionController'
+import PersonalInterestsView from './personalInterestsView'
+
+/**
+ * Abstract controller class defining functionality common to both the Create and Update Induction journeys.
+ */
+export default abstract class PersonalInterestsController extends InductionController {
+  constructor() {
+    super()
+  }
+
+  /**
+   * Returns the Personal Interests view; suitable for use by the Create and Update journeys.
+   */
+  getPersonalInterestsView: RequestHandler = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+    const { prisonerSummary } = req.session
+
```

Comment:
We will need to define and add a form object here, but for the time being this is enough to allow Adam to get the basic nunjucks view working.